common.mk

```diff
@@ -108,6 +108,7 @@ $(FIRRTL_FILE) $(ANNO_FILE) &: $(SCALA_SOURCES) $(sim_files) $(SCALA_BUILDTOOL_D
 		--name $(long_name) \
 		--top-module $(MODEL_PACKAGE).$(MODEL) \
 		--legacy-configs $(CONFIG_PACKAGE):$(CONFIG) \
+		$(if chipyard.upf.ChipTopUPFAspect, --with-aspect chipyard.upf.ChipTopUPFAspect,) \
```

Comment:
This `if` statement will always return as `true` resulting in the `--with-aspect` statement always being added. For now, I would just remove the `if` part of this code block.